experiment_helpers/check_rmsThresh.m

```diff
@@ -7,72 +7,62 @@
 %
 % Input arguments:
 %   * data. The output data structure from Audapter
-%   * rmsThresh. If the RMS value of data is below rmsThresh, the output
-%       parameter bGoodTrial will be 0.
+%   * params. Can be a struct (new format) or double (historical format).
+%       If `params` is a STRUCT: It can contain the following fields
+%         which affect the amplitude calculation:
+%           * checkMethod. If 'peak_window', the RMS value is the mean RMS
+%               during a period centered on the peak. If 'peak',
+%               the RMS value is the absolute peak RMS.
+%           * limits. A 2x2 array structured like this:
+%                  [GoodLow, GoodHi;
+%                   WarnLow, WarnHi]
+%              In check_rmsThresh, an area is shaded green between the low
+%              and hi Good limits, and an area is shaded yellow between Warn limits.
+%          * rmsThresh. If the RMS value is below rmsThresh, the output
+%              parameter bGoodTrial will be 0.
+%          * peakWindowSecs. 
+%       If `params` is a DOUBLE: params will be interpreted as
+%         the value of the field rmsThresh (see above).
 %   * subAxis. If a graphics object is included in the 3rd input parameter,
 %       the mean RMS and OST values are plotted.
-%   * params: A structure (often stored in expt.amplcalc)
-%       controlling parts of this function, with relevant fields:
-%       * checkMethod. If 'mean', the RMS value is calculated as the mean
-%           RMS during the vowel. If 'peak', the RMS value is the peak RMS.
-%       * limits. A 2x2 array structured like this:
-%               [GoodLow, GoodHi;
-%               WarnLow, WarnHi]
-%           In check_rmsThresh, an area is shaded green between the low
-%           and hi Good limits, and an area is shaded yellow between Warn limits.
-%       * rmsThresh. If the RMS value is below rmsThresh, the output
-%           parameter bGoodTrial will be 0. This parameter is overridden by
-%           the 2nd input param `rmsThresh`, if that input param is used.
 %
-% 
 
-if nargin < 3, subAxis = []; end
-if nargin < 4
+if nargin < 2
+    params.checkMethod = 'peak';
```

Comment:
It looks like there's a different default for checkMethod depending on whether you a) don't provide a params argument (defaults to 'peak') or b) provide one with no checkMethod field (defaults to 'peak_window'). This seems confusing; why do we want this behavior?

Reply:
Will remove this to only have one default.